code/modules/recycling/conveyor.dm

```diff
@@ -409,7 +409,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 		return TRUE
 
 /obj/machinery/conveyor_switch/multitool_act(mob/living/user, obj/item/I)
-	var/input_speed = tgui_input_number(user, "Set the speed of the conveyor belts in seconds", "Speed", conveyor_speed, 20, 0.2)
+	var/input_speed = tgui_input_number(user, "Set the speed of the conveyor belts in tenths of a second", "Speed", conveyor_speed, 50, 2) / 10
```

Comment:
The fact that the game runs in tenths of a second is an implementation detail we shouldn't expose to players.

Make this seconds and just set `round_value = FALSE`.